streams/api/src/main/java/org/eclipse/microprofile/reactive/streams/ProcessorBuilder.java

```diff
@@ -199,9 +194,6 @@
      * @throws IllegalArgumentException If {@code n} is less than zero.
      */
     public ProcessorBuilder<T, R> skip(long n) {
-        if (n < 0) {
-            throw new IllegalArgumentException("Cannot skip less than zero elements");
-        }
```

Comment:
Should be these tests made by the stage implementation?
It does not seems to be checks by the TCK (because I don't have the tests)

Reply:
Nevermind, saw them below.